Enable GCP Flow Logs (#535)
gardener/gardener-extensions: controllers/provider-gcp/docs/usage-as-end-user.md

The specified CIDR ranges must be contained in the VPC CIDR specified above, or the VPC CIDR of your already existing VPC.
You can freely choose these CIDRs and it is your responsibility to properly design the network layout to suit your needs.

> The `networks.flowLogs` section describes the configuration for the vpc flow logs. In order to enable the vpc flow logs at least one of the following parameters needs to be specified in the flow log section.

Reviewer: ```suggestion
The `networks.flowLogs` section describes the configuration for the VPC flow logs. In order to enable the VPC flow logs, at least one of the following parameters needs to be specified in the flow log section:
```

> The `networks.flowLogs.aggregationInterval` is optional and describes the aggregation interval for collecting flow logs.

Reviewer: ```suggestion
* `networks.flowLogs.aggregationInterval`, an optional parameter describing the aggregation interval for collecting flow logs.
```

> 

Reviewer: ```suggestion
```

> The `networks.flowLogs.flowSampling` is optional and sets the sampling rate of VPC flow logs within the subnetwork where 1.0 means all collected logs are reported and 0.0 means no logs are reported.

Reviewer: ```suggestion
* `networks.flowLogs.flowSampling`, an optional parameter setting the sampling rate of VPC flow logs within the subnetwork where `1.0` means all collected logs are reported and `0.0` means no logs are reported.
```

> 

Reviewer: ```suggestion
```

> The `networks.flowLogs.metadata` is optional and describes whether metadata fields should be added to the reported VPC flow logs.

Reviewer: ```suggestion
* `networks.flowLogs.metadata`, an optional parameter describing whether metadata fields should be added to the reported VPC flow logs.
```
